### ISSUE TITLE
Charts: visual accessibility of data labels

### ISSUE THREAD

**Issue by andrewleith** [Budibase/budibase#5199]
**Describe the bug**
When cycling through the chart palettes, I noticed many label colors are not accessible.  Not only are they not accessible, many can't be read by anyone no matter how good their eyesight is :)

**Screenshots**
A couple of examples:
- In palette 1 
- #505050 on #319cfb
- Contrast of 2.79:1, fails WCAG AA and AAA
  <img width="59" alt="image" src="https://user-images.githubusercontent.com/823749/161183943-73df8507-d6a0-4f0d-b29b-beebf68daf0f.png">

- In palette 4
- #505050 on #667c88
- Contrast of 1.84:1, fails WCAG AA and AAA
  <img width="118" alt="image" src="https://user-images.githubusercontent.com/823749/161183842-b8c4a69d-6171-489f-97fd-52760fc05db5.png">

**Additional context**
Do you have the palette colors documented somewhere?  Do the palettes change based on theme (light/lighter/dark/darker)?

**Comment by aptkingston** [comment]
Hey @andrewleith. The issue here is that we set the text color (for the whole chart) depending on the theme used. So for light themes we set dark text, which is currently also used for those labels as you've pointed out.

I've had a quick look through the palette options and setting the label color to white should work quite well for them all, so I think that's the fix we'll go for.

**Comment by andrewleith** [comment]
Thanks @aptkingston.  That will probably work in most cases.  I see at least one exception in my current app, in palette 4.  The first two label backgrounds are #61d1c9 and #ccf374 - these won't pass contrast with white text.  If you can only set one font color for all, the background colors may need to be revisited.
